### PR TITLE
feat: add meta morpho vault cap automator

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

File: test/web3-functions/mainnet/meta-morpho.test.ts [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    })
+
+    it('no pending caps', async () => {
+        const { result } = await metaMorphoW3F.run('onRun', { userArgs })
+
+        expect(result.canExec).to.equal(false)
+        !result.canExec && expect(result.message).to.equal('No pending caps to be accepted')
+    })
+
+    it('no pending caps (waiting for the timelock)', async () => {
```
Comment (lucas-manuel) [inline test/web3-functions/mainnet/meta-morpho.test.ts:103]: Boundary condition with timelock needed

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

File: test/web3-functions/mainnet/meta-morpho.test.ts (hunk 1 above)
Comment (barrutko) [reply test/web3-functions/mainnet/meta-morpho.test.ts:103]: Updated

**Review by lucas-manuel — APPROVED** [review]
